eng/pipelines/templates/variables/vmr-build.yml

```diff
@@ -85,6 +85,19 @@ variables:
   - name: signEnabled
     value: false
 
+# If signing is disabled, don't sign the DAC.
+- ${{ if eq(parameters.desiredSigning, 'Unsigned') }}:
```

Comment:
I think I would base this on signEnabled rather than the desired signing parameter. Then you get the automatic translation for default state to true/false. If false, turn off. If true, then further check against the branches like you did below.

It's probably about the same today, but would work more seamlessly if we ever had a feature/* branch.

Reply:
I tried basing it on signEnabled, but the YAML expansion didn't expand it right for it to be respected.